lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,56 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        cube.add_dim_coord(
+            DimCoord(standard_name='time',
+                     units=Unit('days since 0001-01-01 00:00:00',
+                                calendar='gregorian'),
+                     points=np.arange(2),),
+            0)
```

Comment:
Formatting on this makes it difficult to read, instead:

```Python
    def create_cube(self):
        cube = Cube(np.zeros((2, 3, 4)))
    	tunit = Unit('days since 0001-01-01 00:00:00', calendar='gregorian')
	    tcoord = DimCoord(np.arange(2), standard_name='time', units=tunit)
        cube.add_dim_coord(tcoord, 0)
        ...
```